### DISCUSSION TITLE
Welcome to Accucopy Discussions!

### DISCUSSION THREAD

**Discussion by polyactis** [polyactis/Accucopy#7] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JuhoVaan** [comment]
Hi!
I would be interested in testing Accucopy with some of our data and compare the results with Sequenza.
However it seems there are some issues with https://www.yfish.org sites and I can't reach any sites so that I could get my hands on pre-packaged human genomes, manuals etc.
Is there something I'm missing or is there some way to get access to these resources?

**Comment by polyactis** [comment]
Apologies. Site was attacked. It's now in an old version (2019). Trying to
restore it to the latest.

Meanwhile,  two versions of the ref genome are here:

- https://www.yfish.org/static/hs38d1.tar.gz

- https://www.yfish.org/static/hs37d5.tar.gz

--
Yu S. Huang
http://www.yfish.org/
https://sites.google.com/site/polyactis/
